Wikipedia/Code/WMFLanguageCell.m

```diff
@@ -70,14 +73,28 @@ - (void)awakeFromNib {
     [self prepareForReuse];
     [self wmf_makeCellDividerBeEdgeToEdge];
     self.showsReorderControl = YES;
+    self.primaryLabel.textColor = [UIColor whiteColor];
+    self.isPrimary = NO;
 }
 
 - (void)prepareForReuse {
     [super prepareForReuse];
     self.languageNameLabel.text           = @"";
     self.articleTitleLabel.text           = @"";
     self.localizedLanguageLabel.text      = @"";
-    self.languageNameLabelHeight.constant = 0.f;
+    self.localizedLanguageLabelHeight.constant = 0.f;
+    self.isPrimary = NO;
```

Comment:
Probably not now that we're not showing both preferred and non-preferred in the same table any more, but I'll add it just in case.